src/components/invite-dialog/index.tsx

```diff
@@ -77,6 +78,14 @@ export class InviteDialog extends React.Component<Properties, State> {
             alt='Hands reaching out to connect'
             className={c('image')}
           />
+
+          {this.props.isUserInFullScreenModeAndInWorlds && (
+            <Alert variant='info' className={c('network-alert')}>
+              This invite will add someone to your direct messages, <b>not</b> your current network. If you would like
```

Comment:
Take out this second sentence. There's no way to invite someone to a network. (see the first note in the Jira story)

Reply:
done